src/privacy_analysis/system_analysis/system_privacy_root_password.py

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env python3
```

Comment:
Check whether shebang is necessary for class definitions without any "runnable" script

Reply:
I think it's generally just good practice to show what version we're using and have a header for what language we have, even if it's not necessary I think it'd be weird to have the file without it.

Reply:
Yeah but they're not scripts in this case they're just class definitions. I won't hold up the PR about it thought.